.env.example

```diff
@@ -1,4 +1,5 @@
 DATABASE_URL=postgresql://john:start@127.0.0.1:5432/oevent
+SERVER_URL=http://localhost:5000
```

Comment:
This created a lot of problems for us in the past

Reply:
But for accessing the default system image (`header.png` as suggested by you). I'll require this

Reply:
https://stackoverflow.com/questions/15974730/how-do-i-get-the-different-parts-of-a-flask-requests-url

Reply:
`flask.request.url_root` is not letting pass the travis build.
Failed due to that : https://travis-ci.org/bhaveshAn/open-event-server/builds/395595002
Passing if using localhost:5000 : https://travis-ci.org/fossasia/open-event-server/builds/395581727

Reply:
This is not some global module request from flask. This is the request coming to the server. Did you test the implementation?

Reply:
@niranjan94 Please help to fix travis .